cromshell

```diff
@@ -1372,6 +1378,29 @@ if ${ISINTERACTIVESHELL} ; then
   # Check the remaining arguments for help and display it if we have to: 
   checkForComplexHelpArgument $@
 
+  # Get flags:
+  while getopts ":t:" opt ; do
+    case ${opt} in 
+      t)
+        # Ensure that the value given to -t is numerical:
+        echo ${OPTARG} | grep -q '^[0-9][0-9]*$' 
```

Comment:
Is this the same as `^[0-9]+$`?

Reply:
Yes, `grep` doesn't support the `+` operator without also supplying the `-E` operator.

I'll just add `-E`.

Reply:
Ah, I didn't realize that.  Either way is fine.